### PR TITLE
Update finished most back-end and readmes still need to complete front

### PR DESCRIPTION
**Pull request by thorbaert**
No UI updates but mainly build out all structure of the back-end

Note this added the database which is hosted via docker so we will need everyone to have docker installed

### CODE REVIEW

**Review by Gobsmack90 — APPROVED** [review]
I'm approving but lets get my pr through first.

File: server/src/controllers/email.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const db = global.myDb;
+
+router.post('/', async (req, res) => {
+    let body = req.body;
+    try {
+        let result = await db.email.insertOne(body.email);
+        console.log(result);
+        res.sendStatus(200);
+    } catch (e) {
+        if (/already\sexists/.test(e.detail)) {
```
Comment (AvocadoCasualty) [inline server/src/controllers/email.js:12]: Is this a typo? sexists vs exists? ex:

if (/already\exists/.test(e.detail)){

Comment (thorbaert) [reply server/src/controllers/email.js:12]: \s is the character for space

Comment (thorbaert) [reply server/src/controllers/email.js:12]: so what I'm testing for is "already exists" but specficially saying a \s (space) between

**Pull request merged**